frontend/frontend/src/components/Profile.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+	<div>
+		<h2> {{this.$route.params.gameWon==true? "Congratulations": "Better Luck Next Time!"  }}</h2>
+		<h3> Time Stopped at: {{this.$route.params.timer }} </h3>
+		<h3> Match Count: {{ this.$route.params.matches }} </h3>
+
+    </div>
+</template>
+
+<script>
+export default {
+	name: "Profile",
+	data() {
+		return {};
+	},
+
+};	
+</script>
+
+<style scoped>
```

Comment:
I'm wondering if the "done game" page should be different from the profile. I think the the page it routes to be a component called like "GameSummary" that when it's closed they go to their profile/home page. I see the profile building in complexity. so i guess really just naming convention of this

Reply:
Good point, for now since we don't have a profile it seemed right because thats where they see their scores and stuff but Game Summary is definitely better. Changed!